src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/command.html.twig

```diff
@@ -0,0 +1,249 @@
+{% extends '@WebProfiler/Profiler/layout.html.twig' %}
+
+{% block menu %}
+    <span class="label">
+        <span class="icon">{{ source('@WebProfiler/Icon/command.svg') }}</span>
+        <strong>Console Command</strong>
+    </span>
+{% endblock %}
+
+{% block panel %}
+    <h2>
+        {% set command = collector.command %}
```

Comment:
What happens if we try to render the command panel for a request profile ?

Reply:
This cannot happen thanks to this line https://github.com/symfony/symfony/pull/47416/files#diff-84bdbbc00a80f6a3151d87276c235af2f6b90d8ad63da4b648a8a09d980a9b8fR46.